docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -387,6 +388,25 @@ height={1638}
 </TabItem>
 </TabGroup>
 
+### Limiting op concurrency across runs <Experimental />
+
+Limits can be specified on the Dagster instance using the special op tag `dagster/concurrency_key`. If this instance limit would be exceeded by launching an op/asset, then the op/asset will be queued.
+
+For example, to globally limit the number of running ops touching redshift to 2, the op must be first tagged with the global concurrency key:
+
+```python file=/concepts/ops_jobs_graphs/job_execution.py startafter=start_global_concurrency endbefore=end_global_concurrency
+@op(tags={"dagster/concurrency_key": "redshift"})
+def my_redshift_op():
+    ...
+```
+
+To specify limits on the number of ops/assets tagged with a particular concurrency key, you can either use the `dagster` CLI command or use Dagit:
+
+- To specify a global concurrency limit using the CLI, use `dagster instance concurrency set <concurrency_key> <limit>`
```

Comment:
````suggestion
- To specify a global concurrency limit using the CLI, use: 

   ```shell
   dagster instance concurrency set <concurrency_key> <limit>
   ```
````

Suggesting we put this in a code block so users can copy it with a click. You may need to run `make mdx-format` again if you accept this guy

Also, is `concurrency_key` just the value, like `redshift`?

Reply:
> Also, is `concurrency_key` just the value, like `redshift`?


Yes